### PR TITLE
Random lunch and food profile

### PR DESCRIPTION
**Pull request by Quentin-Macquart**
Hey Heloïse, 
just add the fixes for the PR in this branch.

### CODE REVIEW

File: src/pages/LoginPage/index.jsx [hunk 1]
```diff
@@ -127,12 +125,11 @@ export default function LoginPage({ theme, setTheme }) {
               value={form.password}
               onChange={HandleChangeFormData}
             />
-            <input type="submit" value="Lets go" />
+            <input className="letsgo" type="submit" value="Lets go" />
```
Comment (HeloiseFerla) [inline src/pages/LoginPage/index.jsx]: camelCase

File: src/pages/Meeting/Room/AlreadyBooked/index.jsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ export default function AlreadyBooked({
   useEffect(() => {
     const { id } = reservation;
     axios
-      .get(`${process.env.REACT_APP_API_URL}/meeting/${id}/participants`)
+      .get(`http://localhost:5000/meeting/${id}/participants`)
```
Comment (HeloiseFerla) [inline src/pages/Meeting/Room/AlreadyBooked/index.jsx]: Fix that before

**Review by HeloiseFerla — CHANGES_REQUESTED** [review]

**Review by HeloiseFerla — DISMISSED** [review]

File: src/pages/Meeting/Room/AlreadyBooked/index.jsx [hunk 3]
```diff
@@ -25,6 +25,7 @@ export default function AlreadyBooked({
 
       .then(({ data }) => {
         setParticipants(data);
+        console.log(participants);
```
Comment (Hacene010) [inline src/pages/Meeting/Room/AlreadyBooked/index.jsx]: no console log !

File: src/pages/Meeting/Room/AlreadyBooked/index.jsx [hunk 4]
```diff
@@ -37,6 +38,8 @@ export default function AlreadyBooked({
   };
   return (
     <SAlreadyBooked>
+      {console.log(reservation)}
```
Comment (Hacene010) [inline src/pages/Meeting/Room/AlreadyBooked/index.jsx]: no console log

**Review by Hacene010 — CHANGES_REQUESTED** [review]
some evils

File: src/pages/Food/index.jsx [hunk 5]
```diff
@@ -12,25 +13,34 @@ export default function Food() {
   const [dataLunch, setDataLunch] = useState([]);
   const [dataDesserts, setDataDesserts] = useState([]);
   const [dataDrinks, setDataDrinks] = useState([]);
+  const user = useSelector((state) => state.user);
 
   useEffect(() => {
-    axios.get('http://localhost:5000/food/lunch').then(({ data }) => {
+    api.get(`/food/lunch`).then(({ data }) => {
       setDataLunch(data);
+      console.log(user);
```
Comment (Hacene010) [inline src/pages/Food/index.jsx]: ??? :/

**Review by Hacene010 — CHANGES_REQUESTED** [review]
a repetitive evil :-/

File: src/pages/Food/index.jsx (hunk 5 above)
Comment (Quentin-Macquart) [reply src/pages/Food/index.jsx]: i promise you i deleted before push wtf

Comment (Quentin-Macquart) [reply src/pages/Food/index.jsx]: I re-pushed, it should be ok

Comment (Hacene010) [reply src/pages/Food/index.jsx]: Alright, I think I was reviewing the pr when you pushed

**Review by HeloiseFerla — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**